test/e2e/tests/metrics.spec.js

```diff
@@ -36,18 +34,17 @@ describe('Segment metrics', function () {
           .build(),
         ganacheOptions,
         title: this.test.title,
-        failOnConsoleError: false,
```

Comment:
We no longer need this, since we aren't resetting the server we don't have failed requests in the console